### DISCUSSION TITLE
Exporting highlights file from Kindle mobile app

### DISCUSSION THREAD

**Discussion by abishekmuthian** [hadynz/obsidian-kindle-plugin#114] [Ideas]
Kindle mobile app allows exporting highlights and notes(For non amazon books) as .html albeit individually for each book unlike a single file (My Clippings.txt) of Kindle e-reader.

But Kindle app offers some significant advantages over Kindle e-reader for highlighting like being faster, colors etc. and it's likely used by more people as it doesn't require special hardware.

 Sample highlight,note from Kindle app of downloaded content:
[The Air Force has a plan to make jet fuel out of a - Notebook.zip](https://github.com/hadynz/obsidian-kindle-plugin/files/7563524/The.Air.Force.has.a.plan.to.make.jet.fuel.out.of.a.-.Notebook.zip)

Considering the plugin supports Amazon sync, My Clippings.txt I think .html from Kindle app could be supported for feature parity across all platforms.

**Comment by hadynz** [comment]
I'll keep an eye on this thread and see if there many upvotes and will then consider it as an enhancement.

**Reply by simnys** [reply]
Yes! Been looking for a plugin that does this. I think it would provide a lot of value for readers of non-amazon books and people who don't want to plug in their kindle each time to fetch the clippings.txt file.

**Comment by hadynz** [comment]
Can I clarify my understanding of the workflow that users who export to HTML use?

If you are using your tablet as your primary reading device, where do you export your notes to? How/when do you want to import these notes into the Obsidian vault?

**Reply by simnys** [reply]
Of course! Notes and highlights for non official Amazon books made on Kindle tablet still syncs to the Kindle iOS / Android app. From there you can export everything through mail which removes the need for the manual transfer of clippings.txt file.
Hope that make things clearer!

**Reply by OogieM** [reply]
I buy books from sources other than Amazon and also get a lot of free books from Project Gutenberg. I've never been able to get highlights or notes out of any of those books at all. I use the Kindle app on iPad OS to read them so I'd first like to see how to do it and second want a way to save them somehow to Obsidian. There is no share option on those books which is how some people sned the highlights out via html. I want to at the end of reading a book save all the highlights with colors and all notes out to a markdwn note just like it already formats for books from Amazon. My only request is for changes in how to create the filename for the books. I want the tools used in Zotero to define the name, specifically using the author last name rather than the full name in the title.
